backend/store/principal.go

```diff
@@ -219,25 +223,20 @@ func (*Store) listUserImpl(ctx context.Context, tx *Tx, find *FindUserMessage) (
 			&userMessage.PasswordHash,
 			&mfaConfigBytes,
 			&userMessage.Phone,
-			&roles,
 		); err != nil {
 			return nil, err
 		}
 
-		// pgtype cannot assign []string to []api.Role
-		var rolesString []string
-		if err := roles.AssignTo(&rolesString); err != nil {
-			return nil, errors.Wrapf(err, "failed to scan roles")
-		}
-		for _, r := range rolesString {
-			userMessage.Roles = append(userMessage.Roles, api.Role(r))
+		if b := binding; b != nil {
```

Comment:
I'm wondering if we can use SQL to do this...

Reply:
I wanna get rid of Roles from user message entirely. Separate users from workspace policy, completely independent stuff.